README.md

```diff
@@ -70,6 +70,7 @@ can be useful to know which optimizations we apply. This is a list:
   useful in combination with minification, which removes unused code paths.
 - __brfs:__ Statically inline calls to `fs.readFile()`. Useful to ship assets
   in the browser.
+- __babelify:__ Bring the latest browser features to _all_ browsers.
```

Comment:
Can we (should we) put a note about the regeneratorRuntime/babel-polyfill for IE 11 users if you're using `async/await` or generator functions?

It's super hard to find good information about this online (there's a few github tickets on non-babel repos that explain it)